src/main/java/frc5124/robot2020/RobotContainer.java

```diff
@@ -172,6 +193,11 @@ public void disabledInit() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return null;
+    return new ZoomZoom(TrajectoryGenerator.generateTrajectory(
```

Comment:
has this been tested? Is it a real autonomous option? if not remove it from the PR.